### ISSUE TITLE
Will Skeleton continue to be updated now that Bootstrap 2.0 is out?

### ISSUE THREAD

**Issue by schwanksta** [dhg/Skeleton#69]

**Comment by matthewkimber** [comment]
Why would the release of another framework affect Skeleton?

**Comment by gnugat** [comment]
Because the main developer of Skeleton works at Twitter, which is in turn the main developer of Bootstrap (the version 2.0 brings responsiveness).
I should say that I'm worried too that, for this reason, Skeleton might not be updated ever again :/ .

**Comment by dhg** [comment]
Hey fellas - 

Sorry this question took so long to answer as it is a very important one. I've shared this before in other threads and via email, but it's never been directly addressed. I developed Skeleton to be lightweight and simple. It was never intended to become a framework as robust as Bootstrap, because then why not just use Bootstrap? Skeleton was designed to be an easy to use, extensible starting point for responsive projects. Since I usually design from scratch, Skeleton is actually a quicker, more useful starting point for me than more comprehensive projects like Bootstrap or Foundation.

In short, Skeleton will absolutely continue to be maintained and may have small future feature additions, but it will not continue to rapidly grow and receive massive iterations like Bootstrap. Keeping Skeleton lean was always my intention.

Thanks so much for the question and I hope the answer is clear and direct, but if you have any other questions please feel free to hit me up.

**Comment by edhedges** [comment]
As a web developer that does very little design it is everything you need to get a good looking and responsive site up. 

I love that is is light-weight, well documented, and easy to extend/modify. For me (focusing on back end work) there isn't anything better than skeleton in terms of styling pages.

**Comment by schwanksta** [comment]
Thanks for the answer, Dave. Glad to hear it.

**Comment by jeffkee** [comment]
I prefer the lightweight & flexible skeleton as a pure css grid only with no JS involved. I prefer to handle my own JS elements through jquery & jqueryUI and other plug-ins. 

Less is more sometimes. While all platforms have their uses, for the heavier coders who want more control, a CSS-only grid system like this is much more useful and flexible in the long term. For projects that  need quick propagation of simple UI elements etc. very fast, I can see how Bootstrap can help. But I will stay loyal to Skeleton for the time being.
